### DISCUSSION TITLE
Opt-in to being contacted for followups and interviews?

### DISCUSSION THREAD

**Discussion by LeaVerou** [Devographics/surveys#216] [General Survey Design]
Currently, the survey asks for respondent email, but only to notify for future surveys, and does not store it:

<img width="824" alt="image" src="https://github.com/Devographics/surveys/assets/175836/6f95a12a-6769-453f-a766-45bfad01bfdd">

I wonder if it would be nice if respondents could *opt in* to being contacted in the future, either to follow up on some of their comments, or even for interviews. That could be incredibly useful for future research.

Something like:

> ### Optional email
> 
> (Text field)
> 
> **🔒 Your email will not be stored in our server** 
> 
> - [x] Email me when results are published
> - [ ] You can contact me for followups to my comments in the survey
> - [ ] I’m interested in being interviewed about my use of web technologies

The sentence "🔒 Your email will not be stored in our server" would change to "🔓 Your email _will_ be stored in our server" if they select any of the 2nd or 3rd checkboxes.

**Comment by SachaG** [comment]
I could see how that would be useful, but it would force us to link a person's data with their identity (in the form of their email), which we don't want to do to avoid any security issues or data leaks. So it probably won't happen for now.
